### DISCUSSION TITLE
Next major release date

### DISCUSSION THREAD

**Discussion by apdesai16** [Comcast/react-data-grid#3525] [General]
I am currently working on a project that heavily relies on `react-data-grid`, and I'm eager to know about any upcoming release or update which supports react 18.x.x.

**Comment by sknutsonsf** [comment]
V7 claims to support react 18 in the documentation
However, it is still in "canary" status after more than a year.

When will an official "7.0" release come out?

**Comment by mikkelking** [comment]
I'm at the point where I abandon this component, due to the extend time in beta with very little activity.
